### ISSUE TITLE
[Libretro] [Android] Some games that used to run now crash Retroach when opening.

### ISSUE THREAD

**Issue by demodanza** [hrydgard/ppsspp#13803]
### What happens?
Using the latest core on Android Retroarch, some games that used to work some time ago, now crash the app. Some of the games tested that exhibit this issue: Puyo Puyo 20th Anniversary, Dissidia 012 Final Fantasy, Gladiator Begins, Yu-Gi-Oh! ARC-V Tag Force Special. Meanwhile, other games still load fine, like Puyo Puyo 7, Madden NFL 12, etc.
The same games that are crashing run normally on Windows , no crashes there.
Log attached for reference.
[retroarch.log](https://github.com/hrydgard/ppsspp/files/5733041/retroarch.log)

### What *should* happen?
The games should load normally, as they did before.

### What hardware, operating system, and PPSSPP version?  On desktop, GPU matters for graphical issues.
Mipad 4, MIUI 10.3.2
Android 8.1.0
3 GB ram, Octa-core max 2.20 GHz, Armv8
RetroArch 1.9.0 (Git 2ba2e00)
Latest PPSSPP core offered by the buildbot.

**Comment by unknownbrackets** [comment]
Is this similar to #13737?

Does the latest proper PPSSPP version work fine?  What's the last version that worked, and the first version that didn't?  Have you tried other graphics backends?

-[Unknown]

**Comment by demodanza** [comment]
No, the screen does not go black, the whole retroarch client crashes and it happens in both Android and Windows and yes, those games work on standalone.
As far as which core version didn0t have the problem, I couldn't tell, maybe they never worked and I was mixing it with standalone in my mind.

**Comment by unknownbrackets** [comment]
Sounds like this may be fixed now?

-[Unknown]

**Comment by Denshadego** [comment]
Yes, don’t know how, but tested again on both platforms and those games do run! Thanks for keeping support for both stand-alone and libreto!
